pkg/engine/image.go

```diff
@@ -0,0 +1,87 @@
+package engine
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/containers/podman/v4/pkg/bindings/images"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"k8s.io/klog/v2"
+)
+
+const imageMethod = "image"
+
+// Image configures targets to run a system prune periodically
+type Image struct {
+	CommonMethod `mapstructure:",squash"`
+	// Url is the url of the image to be loaded onto the system
+	Url string `mapstructure:"url"`
+}
+
+func (i *Image) GetKind() string {
+	return imageMethod
+}
+
+func (i *Image) SchedInfo() SchedInfo {
+	return SchedInfo{
+		schedule: i.Schedule,
+		skew:     i.Skew,
+	}
+}
```

Comment:
The individual SchedInfo fn isn't necessary, you can remove that, https://github.com/containers/fetchit/pull/194  (SchedInfo is the same for all targets, so it's a method of the CommonMethod type only, and not the interface)